src/PKSim.Presentation/Services/CompoundProcessTask.cs

```diff
@@ -180,10 +180,15 @@ private void updateLipophilicity(CompoundProcess newProcess, Compound compound)
       private void updateProcessParameterFromDefaultAleternative(CompoundProcess newProcess, Compound compound, string processParameterName, string compoundParameterName, string groupName)
       {
          var processParameter = newProcess.Parameter(processParameterName);
-         if (processParameter == null) return;
+         if (processParameter == null)
+            return;
+
          var parameterAlternative = compound.ParameterAlternativeGroup(groupName).DefaultAlternative;
-         if (parameterAlternative == null) return;
+         if (parameterAlternative == null)
+            return;
+
          processParameter.Value = parameterAlternative.Parameter(compoundParameterName).Value;
+         processParameter.IsDefault = false;
```

Comment:
This code also set the is default to false when compound parameter are updated (Lipo, fu exp etc..)